packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -969,6 +970,75 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(map2.get(testKey), testValue);
 	});
 
+	it.skip("ensure single attach op sent in map", async function () {
+		let idB;
+		const cb = async (container, d?) => {
+			const defaultDataStore = (await container.getEntryPoint()) as ITestFluidObject;
+			const runtime = defaultDataStore.context.containerRuntime;
+
+			const dataStoreB = await runtime.createDataStore(["default"]);
+			const dataObjectB = (await dataStoreB.entryPoint.get()) as ITestFluidObject;
+			idB = dataObjectB.context.id;
+
+			defaultDataStore.root.set("B", dataObjectB.handle);
+		};
+
+		await cb(container1);
+		await provider.ensureSynchronized();
+
+		const container3: IContainerExperimental =
+			await provider.loadTestContainer(testContainerConfig);
+		await waitForContainerConnection(container3);
+		const default3 = (await container3.getEntryPoint()) as ITestFluidObject;
+
+		const handleB = default3.root.get("B");
+		const dataObjectB3 = await handleB.get();
+		assert(dataObjectB3.context.id === idB);
+	});
+
+	it.skip("ensure single attach op sent in all other ddss", async function () {
+		let idB;
+		const cb = async (container, d?) => {
+			const defaultDataStore = (await container.getEntryPoint()) as ITestFluidObject;
+			const runtime = defaultDataStore.context.containerRuntime;
+
+			const cellRoot = await defaultDataStore.getSharedObject<ISharedCell>(cellId);
+			const dirRoot = await defaultDataStore.getSharedObject<ISharedDirectory>(directoryId);
+			const stringRoot = await defaultDataStore.getSharedObject<SharedString>(stringId);
+
+			const dataStoreB = await runtime.createDataStore(["default"]);
+			const dataObjectB = (await dataStoreB.entryPoint.get()) as ITestFluidObject;
+			idB = dataObjectB.context.id;
+
+			cellRoot.set(dataObjectB.handle);
```

Comment:
i think we need a test per dds, as grouping them could miss problems, as if any dds binds the handle it will work right, so we miss broken dds.

i would imagine this test using a for loop to run the same test for each dds. maybe the loop would run over an array of funtion that take the defaultDataStore, and store in the dds like:

(defaultDataStore, handle)=>{
const cellRoot = await defaultDataStore.getSharedObject<ISharedCell>(cellId);
cellRoot.set(handle);
}

Reply:
Also, my impression was that the interesting part was not storing `dataObjectB.handle`, but rather the handle from B's `root` to the other DDSes in B.  So the test case would be to store a handle from a particular DDS to some other DDS - all in a DataStore that is currently detached.